File: JavaScript/6kyu__ordinal_numbers.js
/*
Title: Ordinal Numbers
Rank: 6 kyu
*/

// My Solution

function ordinal(number, brief) {
  let str = String(number);
  if (str[str.length - 2] === "1") return "th"
  if (str[str.length - 1] === "1") return "st";
  if (str[str.length - 1] === "2") return brief ? "d" : "nd";
  if (str[str.length - 1] === "3") return brief ? "d" : "rd";
  return "th";
}

File: CSharp/6kyu__ordinal_numbers.cs
/*
Title: Ordinal Numbers
Rank: 6 kyu
*/

// My Solution

using System;
public class Kata
{
  public static string Ordinal(int number, bool brief = false)
  {
    string str = number + "";
    if (str.Length >1 && str[str.Length-2] == '1') return "th";
    if (str[str.Length-1] == '1') return "st";
    if (str[str.Length-1] == '2') return brief ? "d" : "nd";
    if (str[str.Length-1] == '3') return brief ? "d" : "rd";
    return "th";
  }
}